src/actionbar/index.css

```diff
@@ -0,0 +1,53 @@
+@import '../commons/index.css';
+
+:root {
+  --spectrum-actionbar-height: var(--spectrum-global-dimension-size-600);
+  --spectrum-actionbar-padding-x: var(--spectrum-global-dimension-size-200);
+  --spectrum-actionbar-margin-x: var(--spectrum-global-dimension-size-200);
+  --spectrum-actionbar-offset-y: var(--spectrum-global-dimension-size-200);
+
+  --spectrum-actionbar-min-width: 280px;
+  --spectrum-actionbar-max-width: 1280px;
+}
+
+/* topdoc
+{{ actionbar/actionbar.yml }}
+*/
+.spectrum-ActionBar {
+  align-items: center;
+  flex-direction: initial;
+  justify-content: space-between;
+
+  height: var(--spectrum-actionbar-height);
+  padding: 0 var(--spectrum-actionbar-padding-x);
+  min-width: var(--spectrum-actionbar-min-width);
+  max-width: var(--spectrum-actionbar-max-width);
+  width: 100%;
+}
+
+/* topdoc
+{{ actionbar/table-actionbar.yml }}
+*/
+.spectrum-ActionBar-container {
```

Comment:
Since this is an outer component and not a child of ActionBar, what if we made this be the actual `.spectrum-ActionBar`, and changed the existing `.spectrum-ActionBar` to be `.spectrum-ActionBar-popover` (since it's really meant to modify a popover).

Then we can make this container basically do nothing unless you apply `--sticky` or `--fixed` to it, if that's indeed what we want to happen. 

That would change the markup to:

```
<div class="spectrum-ActionBar spectrum-ActionBar--sticky">
  <div class="spectrum-Popover spectrum-ActionBar-popover is-open ">
    <!-- ... -- >
  </div>
</div>
```

Thoughts?